grbl/servo_pen.c

```diff
@@ -67,12 +66,14 @@ void init_servo()
 
 void pen_up()
 {
-	SERVO_OCR_REGISTER = PEN_SERVO_UP;
+  //printString("Pen Up\n");
```

Comment:
Delete this. If you want to keep it around in case you want to debug, consider doing this:
```
#ifdef DEBUG_PEN
    printString("Pen Up\n");
#endif
```

Then you can just add `#define DEBUG_PEN` if you want to turn it on.